.github/workflows/stackhpc.yml

```diff
@@ -27,13 +22,13 @@ jobs:
         run: |
           set -x
           mkdir ~/.ssh
-          echo "${${{ matrix.cloud }}_SSH_KEY}" > ~/.ssh/id_rsa
+          echo "${arcus_SSH_KEY}" > ~/.ssh/id_rsa
```

Comment:
This replacement is `arcus` and the next is `.stackhpc` - is that correct?

Reply:
Yes